what4-abc/src/What4/Solver/ABC.hs

```diff
@@ -495,6 +485,9 @@ bitblastExpr h ae = do
     ConstantArray{} -> arrayFail
     SelectArray{} -> arrayFail
     UpdateArray{} -> arrayFail
+    CopyArray{} -> arrayFail
```

Comment:
I think these cases are enough to fix #154, so it would be worth mentioning that in the commit message.